main.go

```diff
@@ -43,6 +48,10 @@ var (
 		},
 	)
 	invalidChars = regexp.MustCompile("[^a-zA-Z0-9_]")
+
+	// Udp
+	bindAddress = flag.String("udp.bind-address", ":8089", "Address on which to listen for udp packets.")
```

Comment:
What we've done elsewhere is to default this to the empty string, meaning disabled. In this case it should be safe to default to 9122.